builders/interact/substrate-api/polkadot-js-api.md

```diff
@@ -257,6 +257,31 @@ console.log(`Submitted with hash ${txHash}`);
 
 Note that the `signAndSend` function can also accept optional parameters, such as the `nonce`. For example, `signAndSend(alice, { nonce: aliceNonce })`. You can use the [sample code from the State Queries](#state-queries){target=_blank} section to retrieve the correct nonce, including transactions in the mempool.
 
+### Fee Information {: #fees}
+
+The transaction endpoint also offers a method to obtain weight information for a given `api.tx.<module>.<method>`. To do so, you'll need to use the `paymentInfo` function after having built the entire transaction with the specific `module` and `method`.
+
+The `paymetnInfo` function returns weight information in terms of `refTime` and `proofSize`, which can be used to determined the transaction fee. This is extermely helpful when crafting remote execution calls via XCM.
```

Comment:
```suggestion
The `paymetnInfo` function returns weight information in terms of `refTime` and `proofSize`, which can be used to determine the transaction fee. This is extremely helpful when crafting remote execution calls via XCM.
```

Reply:
doing a drive by 🚗 `paymentInfo` is a typo too!